app/composer.json

```diff
@@ -107,12 +107,18 @@
     "phpoffice/phpspreadsheet": "^1.15",
     "predis/predis": "^1.1",
     "composer/composer": "^2.2",
-    "beberlei/doctrineextensions": "^1.3"
+    "beberlei/doctrineextensions": "^1.3",
+    "freshconsulting/wcag_linter": "dev-master",
+    "squizlabs/php_codesniffer": "^3.6"
```

Comment:
Do we need this in the production dependencies? I think it should easily live in the dev dependencies. It is needed only for development, right?

Reply:
You are right. I'll change.